odl/solvers/functional/default_functionals.py

```diff
@@ -1046,8 +1047,23 @@ def __init__(self, space, prior=None):
         space : `DiscreteLp` or `FnBase`
             Domain of the functional.
         prior : ``space`` `element-like`, optional
-            Data term, positive.
+            Depending on the context, the prior, target or data
+            distribution. It is assumed to be nonnegative.
             Default: if None it is take as the one-element.
+
+        Examples
+        --------
```

Comment:
If you could, adding a short sentence explaining what happens usually lightens up the mood here